src/main/java/com/google/sps/data/SmallCityService.java

```diff
@@ -48,10 +58,18 @@ public void createUserWithGeolocation(MapLocation mapLocation) {
   public void findAllBusinesses() {
     businesses = businessesService.getBusinessesFromPlacesApi(user);
   }
+  
+  // To be used for unit testing file to be able to 
+  // set any static business LinkedList we want to try to use
+  public void setAllBusinesses(List<Listing> allBusinesses) {
+    businessesService.setAllBusinesses(allBusinesses);
+  }
 
-  public void eliminateBigBusinesses() {
-    // TODO: Parse big business list and remove big businesses from businessList
-
+  // To remove the big businesses from the list 
+  // that will be returned from the use of the Places API 
+  public void filterTheListOfBusinessesIntoSmallBusinesses() {
+    databaseResults = businessesService.getBigBusinessFromDatabase();
```

Comment:
Should this be getBigBusinessFromDatabase or getBigBusinessesFromDatabase? 
Essentially this name should reflect whether this is pulling one, or more businesses.

Reply:
since it is pulling the whole database, it is getting all the big businesses in the database. So it should be getBigBusinessFromDatabase().